lightstep/instrument.py

```diff
@@ -52,6 +52,9 @@ def __init__(self,
             warnings.warn('SSL CERTIFICATE VERIFICATION turned off. ALL FUTURE HTTPS calls will be unverified.')
             ssl._create_default_https_context = ssl._create_unverified_context
 
+        if group_name is None:
+            group_name = sys.argv[0]
```

Comment:
`argv[0]` is an ok-but-not-great substitute for an explicit group name... thoughts about a `warnings.warn` call here (like the one a few lines above) about the best-guess default?

If we do that – and, actually, even if we don't – I think this file needs to `import warnings`, btw.